tensorboard/plugins/profile_redirect/tf_profile_redirect_dashboard/tf-profile-redirect-dashboard.html

```
@@ -0,0 +1,110 @@
+  <template>
+    <div class="message">
+      <h3>The profile plugin has moved.</h3>
+      <p>
+        Please install the new version of the profile plugin from PyPI by
+        running the following command from the machine running TensorBoard:
+      </p>
+      <textarea id="commandTextarea" readonly on-blur="_removeCopiedMessage">
+[[_installCommand]]</textarea
+      >
+      <div id="copyContainer">
+        <span id="copiedMessage"></span>
+        <paper-button raised on-tap="_copyInstallCommand"
+          >Copy to clipboard</paper-button
+        >
+      </div>
+    </div>
+
+    <style include="dashboard-style"></style>
+    <style>
+      .message {
+        margin: 80px auto 0 auto;
+        max-width: 540px;
+      }
+      #commandTextarea {
+        margin-top: 1ex;
+        padding: 1ex 0;
+        resize: vertical;
+        width: 100%;
+      }
+      #copyContainer {
+        display: flex;
+      }
+      #copiedMessage {
+        align-self: center;
+        flex-grow: 1;
+        font-style: italic;
+        padding-right: 1em;
+        text-align: right;
+      }
+    </style>
+  </template>
+
+  <script>
+    (function() {
+      Polymer({
+        is: 'tf-profile-redirect-dashboard',
+        properties: {
+          _installCommand: {
+            type: String,
+            readonly: true,
+            value: [
+              'pip install --extra-index-url https://test.pypi.org/simple/ \\',
+              '    tensorboard_plugin_profile',
+            ].join('\n'),
+          },
+        },
+        _copyInstallCommand() {
+          const tryCopy = (async () => {
+            const textarea = this.$.commandTextarea;
+            textarea.select();
+            try {
+              await navigator.clipboard.writeText(textarea.textContent);
+            } catch (error) {
+              // Fallback approach.
+              if (!document.execCommand('copy')) {
+                return Promise.reject();
```

I believe you can throw an error here instead (since this is inside an async block; it effectively is the same but functionally might differ--promise that returns rejected promise vs. promise that rejects).